conf/evolutions/default/1.sql

```diff
@@ -2,7 +2,9 @@
 create table "short_link" (
   "id" bigint generated by default as identity(start with 1) not null primary key,
   "redirect_to_url" varchar not null,
-  "token" varchar not null
+  "token" varchar unique not null,
+  "expiration_date" date,
+  "expired" boolean not null default 0
```

Comment:
Since I currently have the local database starting up along with the service, I was able to get away with editing this script - but in practice, these new columns should have been created a separate migration/evolution script.